### ISSUE TITLE
could jbang-vscode offer to install jbang if it is not available?

### ISSUE THREAD

**Issue by maxandersen** [jbangdev/jbang-vscode#53]
its all nice downloading jbang + jbang edit  gives you full dev environment but
it would be nice if jbang-vscode could offer to install jbang to make it easy to start from downloading vscode instead of installing script via terminal.

wdyt?

**Comment by fbricon** [comment]
Yeah I thought about it, also to embed jbang by default.
Where should we install it by default? ~/.jbang?

**Comment by maxandersen** [comment]
yeah - I was thinking to just do what `curl -Ls https://sh.jbang.dev | bash -s - app setup` does.

for windows its a bit more tricky as it might require elevated privileges due to reliance on symbolic linking ;/

**Comment by maxandersen** [comment]
I would be -1 on embedding jbang by default; unless its just used as absolute last fallback or we can release vscode-jbang immediately when jbang updates.

**Comment by fbricon** [comment]
> yeah - I was thinking to just do what `curl -Ls https://sh.jbang.dev | bash -s - app setup` does.
> 
> for windows its a bit more tricky as it might require elevated privileges due to reliance on symbolic linking ;/

So do you want that we run curl / iex from vscode to trigger the install in a terminal, or do you have something else in mind?

**Comment by fbricon** [comment]
... like detect if sdkman, homebrew, choco is installed and delegate the JBang installation to those
